tests/exchange/test_exchange.py

```diff
@@ -4220,3 +4231,154 @@ def test_get_max_pair_stake_amount(
     mocker.patch('freqtrade.exchange.Exchange.markets', markets)
     assert exchange.get_max_pair_stake_amount('BTC/USDT', 2.0) == 20000
     assert exchange.get_max_pair_stake_amount('ADA/USDT', 2.0) == 500
+
+
+def test_load_leverage_tiers(mocker, default_conf, leverage_tiers):
+    api_mock = MagicMock()
+    api_mock.fetch_leverage_tiers = MagicMock()
+    type(api_mock).has = PropertyMock(return_value={'fetchLeverageTiers': True})
+    default_conf['dry_run'] = False
+    default_conf['trading_mode'] = 'futures'
+    default_conf['margin_mode'] = 'isolated'
+
+    ccxt_exceptionhandlers(
```

Comment:
are you intentionally only testing the exception handlers? (i noticed firstly because `leverage_tiers` is not used)

In theory, we could have the "return" statement missing in the function - and we'd never notice ... 
while now it's a plain "passthrough" of ccxt, nothing guarantees that stays this way ...